lib/nve/nve-main-page.dart

```diff
@@ -295,6 +329,11 @@ class GreenPage extends State<MyStatefulGreenPage> {
   void _setAlerts(bool value) {
     setState(() {
       settings.enableAlerts = value;
+      if (settings.enableAlerts == true) {
+        notificationService.showAlertNotification();
+      } else {
+        notificationService.cancelNotification(0);
```

Comment:
avoid magic numbers like 0. It can be confusing to read numbers with no explaination. Why 0 ? What does it mean ? 
=> Use a constant instead